src/renderer/views/Watch/Watch.js

```diff
@@ -881,6 +940,42 @@ export default defineComponent({
       }
     },
 
+    /**
+     * @param {import('youtubei.js').Misc.Format[]} audioFormats
+     * @returns {AudioSource[]}
+     */
+    createLocalAudioSourceList: function (audioFormats) {
+      return audioFormats.sort((a, b) => {
+        return a.bitrate - b.bitrate
```

Comment:
Why not return `b.bitrate - a.bitrate` so you dont have to reverse it later?

Reply:
@absidue

Reply:
That was the existing logic, I didn't make any changes to it other than removing the arrow function around the switch statement.

I'll take a look if that's possible to do, just need to keep in mind that the list of audio formats varies per video, usually there are 4. For videos that only go up to 480,p there are only the low and medium one and some other videos have way more than 4, so only the 4 lowest bitrate ones get a text label and all the others get a bitrate label.

Reply:
If I do the logic change, I'll add more test cases, so you'll have to test those too, to check that it didn't break existing behaviour.

Reply:
We would have to get rid of the switch statement and do a bunch of stuff based on array length, as we would need to count from the end of the array instead of the start. It's doable, just makes the logic more complicated.

### Current implementation (sort lowest to highest and reverse later):
#### 2 formats:
| Index | Label |
| --- | --- |
| `0` | Low |
| `1` | Medium |

#### 4 formats:
| Index | Label |
| --- | --- |
| `0` | Low |
| `1` | Medium |
| `2` | High |
| `3` | Best |

#### 6 formats:
| Index | Label |
| --- | --- |
| `0` | Low |
| `1` | Medium |
| `2` | High |
| `3` | Best |
| `4` | {bitrate} |
| `5` | {bitrate} |

### Suggestion (sort highest to lowest):
#### 2 formats:
| Index | Label |
| --- | --- |
| `0` | Medium |
| `1` | Low |

#### 4 formats:
| Index | Label |
| --- | --- |
| `0` | Best |
| `1` | High |
| `2` | Medium |
| `3` | Low |

#### 6 formats:
| Index | Label |
| --- | --- |
| `0` | {bitrate} |
| `1` | {bitrate} |
| `2` | Best |
| `3` | High |
| `4` | Medium |
| `5` | Low |

Reply:
@absidue @ChunkyProgrammer could one of u create an issue for this so it doesnt get lost in the GH void

Reply:
No, this is not a change worth making.